src/sonic-frr-mgmt-framework/templates/bgpd/bgpd.conf.j2

```diff
@@ -0,0 +1,22 @@
+!
+! template: bgpd/bgpd.conf.j2
+!
+{% block banner %}
+! =========== Managed by sonic-cfggen DO NOT edit manually! ====================
+! generated by templates/quagga/bgpd.conf.j2 with config DB data
+! file: bgpd.conf
+!
+{% endblock banner %}
+!
+{% include "common/daemons.common.conf.j2" %}
+!
+agentx
+!
+{% if DEVICE_METADATA['localhost']['type'] == "SpineChassisFrontendRouter" %}
```

Comment:
We dont need this check, please remove this.

Reply:
Addressed